src/JsonSchema/Constraints/Factory.php

```diff
@@ -10,15 +10,20 @@
 namespace JsonSchema\Constraints;
 
 use JsonSchema\Exception\InvalidArgumentException;
+use JsonSchema\SchemaStorage;
 use JsonSchema\Uri\UriRetriever;
-use JsonSchema\Validator;
 
 /**
  * Factory for centralize constraint initialization.
  */
 class Factory
 {
     /**
+     * @var SchemaStorage
+     */
+    protected $schemaStorage;
```

Comment:
We've a getter method for $uriRetriever, wouldn't it make sense to have one for `$schemStorage`, too?